generated/nidigital/nidigital/session.py

```diff
@@ -1938,14 +2432,14 @@ def get_history_ram_sample_count(self):
     def get_pattern_name(self, pattern_index):
         r'''get_pattern_name
 
-        TBD
+        You must provide a ViChar array to serve as a buffer for the value. You pass the number of bytes in the buffer as the **bufferSize**. If the current value of the property, including the terminating NULL byte, is larger than the size you indicate in the **bufferSize**, the method copies (bufferSize - 1) bytes into the buffer, places an ASCII NULL byte at the end of the buffer, and returns the **bufferSize** you must pass to get the entire value. For example, if the value is "123456" and the **bufferSize** is 4, the method places "123" into the buffer and returns 7. If you want to call this method just to get the required buffer size, you can pass 0 for the **bufferSize** and VI_NULL for the value.
```

Comment:
Content about ViChar array and buffer doesn't apply in Python. In fact, I think this method shouldn't even be public. Delete the doc content for now. I created #1557 to follow up.

Reply:
Done